_rules/role-required-states-and-properties-4e8ab6.md

```diff
@@ -205,23 +229,18 @@ This `div` does not have a [semantic role](#semantic-role).
 
```

Comment:
I'd rephrase the description of "Inapplicable Example 1" to clarify that it is included here because, owing to its generic role, it does not feature in the accessibility tree.

Conversely, despite lacking a corresponding implicit role in ARIA, an `iframe` element, presumed to lack an implicit semantic role, is included in the accessibility tree, and therefore this rule is applicable.